classes/submission/Submission.inc.php

```diff
@@ -384,14 +401,18 @@ function getAuthorEmails() {
 	/**
 	 * Get all authors of this submission.
 	 * @param $onlyIncludeInBrowse boolean whether to limit to include_in_browse authors.
+	 * @param $version int
 	 * @return array Authors
 	 */
-	function getAuthors($onlyIncludeInBrowse = false) {
+	function getAuthors($onlyIncludeInBrowse = false, $version = null) {
 		$authorDao = DAORegistry::getDAO('AuthorDAO');
-		if (!$onlyIncludeInBrowse)
-			return $authorDao->getBySubmissionId($this->getId());
-		else
-			return $authorDao->getBySubmissionId($this->getId(), false, true);
+		if ($version == null) $version = 1;
```

Comment:
Hmm... shouldn't it be the last submission version? I'll have to think about all those author things properly later... :-)